index.html

```diff
@@ -38,6 +39,156 @@
     </div>
 
   </section>
+
+  <section class="work-section">
+    <div class="RWbox"><div class="RWork"><p>My Recent works</p>
```

Comment:
- [ ] Kindly ensure that the title (e.g., "My Recent Works") uses a header tag `hx` (I would recommend you use the `<h2>` tag). This is so because all `<section>` tag requires a header title hence why you need to use the heading tag( e.g `<h2>` tag) 👍🏽